packages/styles/scss/internal/scroll-into-view/_scroll-into-view.scss

```
@@ -0,0 +1,25 @@
+/**
+ * Copyright IBM Corp. 2016, 2020
+ *
+ * This source code is licensed under the Apache-2.0 license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+@import '../../globals/vars';
+@import '../../globals/imports';
+
+:root {
+  --#{$dds-prefix}--delay: 400ms;
```

My apologies for another name change request, but I figured that the variable should be more specific of what it does (instead of using name implying a general purpose):

```suggestion
  --#{$dds-prefix}--scroll-into-view-delay: 400ms;
```

---

You're right, the more specific the better to avoid any potential conflicts 👍